### ISSUE TITLE
Allow Custom IP to bind to

### ISSUE THREAD

**Issue by lionkor** [BeamMP/BeamMP-Server#57]
Currently you can specify a custom IP via the first argument to the server. It's then subsequently ignored by the backend. 

The entire thing needs to be an optional config setting, and then be bound to with bind() so that we can ensure the IP is actually owned by the server.

**Comment by lionkor** [comment]
Needs to work in docker/pterodactyl, so ensure we can still check the IP by bind()ing to it

**Comment by Fireant456** [comment]
Is this still being worked on?

**Comment by lionkor** [comment]
Yes, but I'm currently taking a bit of a break from the project.

**Comment by lionkor** [comment]
But this is still planned and prioritized

**Comment by lionkor** [comment]
has to be adjusted for rust server
